static/app/views/settings/project/projectKeys/projectKeyCredentials.tsx

```diff
@@ -414,12 +395,10 @@ function ProjectKeyCredentials({
           help={tct('The DSN tells the SDK where to send the events to. [link]', {
             link: showDsn ? (
               <Link
-                to={{
-                  ...location,
-                  query: {
-                    ...location.query,
-                    showDeprecated: showDeprecatedDsn ? undefined : 'true',
-                  },
+                to=""
+                onClick={e => {
+                  e.preventDefault();
+                  setShowDeprecatedDsn(showDeprecatedDsn ? null : true);
```

Comment:
Not sure if this is the best pattern, let me know if there's a better alternative!

Reply:
It's an issue with our testing adapter. I think @scttcper has a PR up to fix it though, maybe we should just merge that one?